xml/System.Numerics/INumberBase`1.xml

```diff
@@ -889,7 +889,7 @@ For <xref:System.Numerics.IFloatingPointIeee754%601> this method matches the IEE
         <param name="y">The value to compare with <paramref name="x" />.</param>
         <summary>Compares two values to compute which has the greater magnitude and returning the other value if an input is <c>NaN</c>.</summary>
         <returns>
-          <paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
+          <paramref name="x" /> if it has the greater magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```

Comment:
replace "the" with "a":
```suggestion
          <paramref name="x" /> if it has a greater magnitude than <paramref name="y" />; otherwise, <paramref name="y" />.</returns>
```